config/locales/doc_auth/es.yml

```diff
@@ -190,7 +189,7 @@ es:
       verify_at_post_office: Verifique su identidad en una oficina de correos
       verify_identity: Verifique su identidad
       verify_online: Verifique su identidad en línea
-      welcome: 'Verifiquemos su identidad para %{sp_name}'
+      welcome: 'Verifiquemos su identidad para%{sp_name}'
```

Comment:
Is the space removed intentionally?

Reply:
Nope. Thanks! I put it [back](https://github.com/18F/identity-idp/pull/10513/commits/e73965297f1c12ec06e6a2fc33397b6a270d9296).